posthog/api/paths.py

```diff
@@ -16,60 +17,61 @@ def _event_subquery(self, event: str, key: str):
 
     # FIXME: Timestamp is timezone aware timestamp, date range uses naive date.
     # To avoid unexpected results should convert date range to timestamps with timezone.
-    def _add_event_and_url_at_position(self, aggregate: QuerySet, team: Team, index: int, date_query: Dict[str, datetime.date], urls: Optional[List[str]]=None) -> QuerySet:
-        event_key = 'event_{}'.format(index)
-
-        # adds event_1, url_1, event_2, url_2 etc for each Person
-        return aggregate.annotate(**{
-            event_key: Subquery(
-                Event.objects.filter(
-                    team=team,
-                    event='$pageview',
-                    distinct_id=OuterRef('distinct_id'),
-                    **date_query,
-                    **{'properties__$current_url__isnull': False},
-                    **({'timestamp__gt': OuterRef('timestamp_{}'.format(index - 1))} if index > 1 else {})
-                )\
-                .exclude(**({'properties__$current_url': OuterRef('url_{}'.format(index -1))} if index > 1 else {}))\
-                .order_by('id').values('pk')[:1]
-            ),
-            'timestamp_{}'.format(index): self._event_subquery(event_key, 'timestamp'),
-            'url_{}'.format(index): self._event_subquery(event_key, 'properties__$current_url')
-        })
-
     def list(self, request):
         team = request.user.team_set.get()
         resp = []
         date_query = request_to_date_query(request.GET)
-        aggregate: QuerySet[PersonDistinctId] = PersonDistinctId.objects.filter(team=team)
 
-        aggregate = self._add_event_and_url_at_position(aggregate, team, 1, date_query)
-        urls: List[str] = []
+        sessions = Event.objects.filter(
+                team=team,
+                event='$pageview',
+                **date_query,
+                **{'properties__$current_url__isnull': False},
```

Comment:
I shaved about 2 seconds of 11 second runtime by removing this. I _think_ that's safe as every pageview really should have a url